cpp/oneapi/dal/algo/logistic_regression/detail/train_ops.cpp

```diff
@@ -64,6 +64,8 @@ struct train_ops_dispatcher<Policy, Float, Method, Task> {
 
 INSTANTIATE(float, method::dense_batch, task::classification)
 INSTANTIATE(double, method::dense_batch, task::classification)
```

Comment:
please add a new line here + there is mismatch between the name of this pr and changes inside, overall lgtm